infinispan-spring-boot-starter-remote/src/test/java/test/org/infinispan/spring/starter/remote/InfinispanJmxConfigurationTest.java

```diff
@@ -0,0 +1,36 @@
+package test.org.infinispan.spring.starter.remote;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.spring.starter.remote.InfinispanJmxConfiguration;
+import org.infinispan.spring.starter.remote.InfinispanRemoteAutoConfiguration;
+import org.infinispan.spring.starter.remote.InfinispanRemoteCacheManagerAutoConfiguration;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jmx.export.MBeanExporter;
+
+import test.org.infinispan.spring.starter.remote.testconfiguration.InfinispanCacheTestConfiguration;
+
+@SpringBootTest(classes = {
+      InfinispanRemoteAutoConfiguration.class,
+      InfinispanRemoteCacheManagerAutoConfiguration.class,
+      InfinispanJmxConfiguration.class,
+      InfinispanCacheTestConfiguration.class,
+      MBeanExporter.class},
+      properties = {
+            "spring.main.banner-mode=off",
+            "infinispan.remote.server-list=180.567.112.333:6668",
+            "infinispan.remote.jmx=true",
+            "infinispan.remote.enabled=true"})
+public class InfinispanJmxConfigurationTest {
+
+   @Autowired
+   RemoteCacheManager remoteCacheManager;
+
+   @Test
+   public void notLoadRemoteCacheManagerJmxBeanWithSpring() {
```

Comment:
I am obviously missing something here, but I don't understand how this is testing the JMX behavior. Could you please elaborate?

Reply:
If JMX is enabled and the configured bean does not exclude 'remoteCacheManager' the test fails due to a JMXInstance exception. The only thing I need to have the exception is running a test with the configuration and injecting the remotecachemanager (and still, even without injecting it fails). I just did not know how to assert things! But test name is just not understandable at all, I smoke something yesterday. I will add comments and another test name

Reply:
Sounds good, thanks :)